index.html

```diff
@@ -4655,6 +4656,7 @@ <h2>Definition of Roles</h2>
 								<li><pref>aria-multiselectable</pref></li>
 								<li><pref>aria-readonly</pref></li>
 								<li><pref>aria-required</pref></li>
+								<li><pref>aria-valuetext</pref></li>
```

Comment:
This idea makes sense for the collabsible listbox where there may be an element representing a listbox that does not have any options. However, I think we should deprecate that pattern in APG. Select-only combobox completely replaces it; the combobox is simpler to code and supports aria-required. In any other listbox, the selected option represents the value.